src/reporting/lambda/custom-resource/quicksight/index.ts

```diff
@@ -406,12 +405,13 @@ const createDataSet = async (quickSight: QuickSight, awsAccountId: string, princ
       },
       LogicalTableMap: needLogicalMap ? logicalMap : undefined,
       ColumnGroups: colGroups.length > 0 ? colGroups : undefined,
-      DatasetParameters: props.datasetParameters,
```

Comment:
update test case to make sure we don't set this field

Reply:
we can remove it from interface DataSetProps to make sure we don't use this parameter